src/clj/fluree/db/query/fql/parse.cljc

```diff
@@ -570,7 +575,7 @@
 (defn parse-analytical-query
   [q]
   (let [context       (context/extract q)
-        [vars values] (parse-values q context)
+        [vars values] (parse-values (:values q) context)
```

Comment:
Was there a problem with passing the entire query to the `parse-values` function? I would prefer to follow that pattern because there are some steps which might need to get information from multiple parts of the query, and I think the parsing code is clearer when each parsing function acts on the query consistently.

Reply:
Not so much a problem as it being more inconvenient. `parse-values` is called in three places:
- `parse-analytical-query`, where passing the entire query is easy
- `parse-txn`, where the values have to be put into an artificially constructed query map
- `parse-pattern :values`, where we would have to artificially construct a query map

On the balance of uses, I thought it was cleaner to refactor it to just accept the values instead of assuming that it comes attached to a query, since most of the time it's actually not.